tests/storage/cdi_import/test_import_http.py

```diff
@@ -360,6 +360,7 @@ def test_successful_import_basic_auth(
     indirect=True,
 )
 def test_wrong_content_type(
+    skip_test_if_no_filesystem_sc,
```

Comment:
Every kubernetes storage class supports Filesystem-RWO. If some tests need specifically FS, we need to create a DV/PVC by explicitly passing `volume_mode` FS

Reply:
no programatic skips should be added